src/main/java/io/spring/api/ArticleFavoriteApi.java

```diff
@@ -62,6 +58,6 @@ public ResponseEntity unfavoriteArticle(@PathVariable("slug") String slug,
 
     private Article getArticle(String slug) {
         return articleRepository.findBySlug(slug).map(article -> article)
-            .orElseThrow(ResourceNotFoundException::new);
+                .orElseThrow(ResourceNotFoundException::new);
```

Comment:
Is this intentional ? If no then try formatting only the lines that you have changed, it will keep PRs short and to the point.